ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1750,11 +1750,11 @@ void GCS_MAVLINK_Copter::handleMessage(mavlink_message_t* msg)
         }
 
         if (!pos_ignore && !vel_ignore && acc_ignore) {
```

Comment:
I don't like adding new functionality and refactoring at the same time.
Remembers Kent Beck's words. "Only Wear One Of Four Hats."